### ISSUE TITLE
Choose r2 in clump_r2

### ISSUE THREAD

**Issue by jamesyao666** [MRCIEU/TwoSampleMR#37]
I found my MR results were changed using different r2.  Pruning SNPs by stringent r2<0.001 , there is usually only one sentinel SNP left and I got significant MR result, but using 0.01, I have several SNPs left, which lead to non-significant MR results. 

Which results I should trust?

**Comment by mightyphil2000** [comment]
Are you using your own exposure or outcome? Ideally need all the code used to generate result so we can reproduce it.

Need to also see individual snp results and heterogeneity statistics.

Get Outlook for iOS<https://aka.ms/o0ukef>

________________________________

**Comment by jamesyao666** [comment]
Thanks for your suggestion.  Yes, I am using my own exposure, but the outcome is from MRBASE GWAS.

I know the c1 is not significant , because two SNPs got opposite direction on outcome. While there is no such problem, because only one SNP left in c2. And the heterogeneity is significant in  c1.   

Does that mean I should use c2 as exposure in this case?  Should I use r2=0.001 for all the clumping SNP?

**Comment by mightyphil2000** [comment]
Can you send the individual snp results and heterogeneity test results.


Get Outlook for iOS<https://aka.ms/o0ukef>

________________________________

**Comment by jamesyao666** [comment]
Individual snp results:
SNP	b	se	p
s1	-4.96094616	1.0211095	1.18E-06
s2	10.53463261	1.9526844	6.85E-08

heterogeneity test:

exposure                    method          Q Q_df       Q_pval
c1                   Inverse variance weighted 49.4503105    1 **2.034603e-12**

**Comment by mightyphil2000** [comment]
Are you sure you the effect alleles are correct? Do both effect alleles reflect higher or lower levels of the exposure?

Get Outlook for iOS<https://aka.ms/o0ukef>

________________________________

**Comment by jamesyao666** [comment]
No, the effect alleles of these two SNPs have opposite effect on exposure. One has effect allele "G" and another has effect allele "C". I think the effect alleles are correct, because they are from the same GWAS results and I did the harmonise with outcome.

**Comment by mightyphil2000** [comment]
Can you share your instruments? SNP-exp effect and se and effect allele , other allele and effect allele frequency

Get Outlook for iOS<https://aka.ms/o0ukef>

________________________________

**Comment by jamesyao666** [comment]
Here is the two SNPs as instruments:

beta	                                se	        pval	     SNP	    effect_allele	other_allele	EAF	SNP_Imputation_RSq
-0.024260865	0.000826395	3.44E-171	s1	G	A	0.60017	0.99942
0.033155895	0.003312945	2.7E-23	     s2	C	T	0.94076	0.31118
---------------

I just noticed the second snp has the low imputation quality 0.3. Is that the problem?
